app/src/main/java/org/simple/clinic/home/patients/illustration/DayOfMonth.kt

```diff
@@ -0,0 +1,30 @@
+package org.simple.clinic.home.patients.illustration
+
+import com.squareup.moshi.FromJson
+import com.squareup.moshi.ToJson
+import org.simple.clinic.util.RoomEnumTypeConverter
+import org.threeten.bp.Month
+import java.util.Locale
+
+data class DayOfMonth(
+    val day: Int,
+    val month: Month
+) {
+
+  class MonthRoomTypeConverter : RoomEnumTypeConverter<Month>(Month::class.java)
+
+  object MoshiTypeConverter {
+    @FromJson
+    fun toModel(value: String): DayOfMonth {
+      val splits = value.split(" ")
+      return DayOfMonth(
+          day = splits[0].toInt(),
+          month = Month.valueOf(splits[1].toUpperCase(Locale.ROOT))
+      )
+    }
+
+    @ToJson
+    fun fromModel(enum: DayOfMonth): String =
+        "${enum.day} ${enum.month}"
```

Comment:
There is an inconsistency between the input and output here. If we consider the string "01 June" and try to parse it, we'll get `DayOfMonth(1, Month.JUNE)`. However, if we serialize `DayOfMonth(1, Month.JUNE)`, we'll get `01 JUNE`.

We should retain consistency between the serialized and deserialized form as must as possible.

Reply:
I think this is fine, as we can just specify 1 June instead of padding it with zeroes, which adds extra overhead IMO.

Reply:
I mean the capitalization of the month.